Program.cs

```diff
@@ -0,0 +1,231 @@
+﻿using System;
+using System.Globalization;
+
+
+namespace DateAndTime
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+
+            //FirstExercise();
+            //NYears();
+            //HumanFriendly();
+            //FiveSeconds();
+            //MiliSeconds();
+            //SixMonths();
+            //LastSecond();
+            //DiffInSeconds();
+            //Age();
+            //Thirty();
+            //TwoDays();
+            //Print();
+            //FiveDays();
+            //FirstMonday();
+            //TwoDates();
+            //Sunday();
+
+
+
+
+
+        }
+        static void FirstExercise()
+        {
+
+            //1.Write a program to display the:
+            //a) Current date and time
+            //b) Current year
+            //c) Month of year
+            //d) Week number of the year
+            //e) Weekday of the week
+            //f) Day of year
+            //g) Day of the month
+            //h) Day of week
+            //j) if the current year is a leap year or not
+
+
+            Console.WriteLine(DateTime.Now);
+            Console.WriteLine(DateTime.Now.Year);
+            Console.WriteLine(DateTime.Now.Month);
+            Console.WriteLine(DateTime.Now.DayOfYear / 7);
+            Console.WriteLine((int)DateTime.Now.DayOfWeek);
+            Console.WriteLine(DateTime.Today.DayOfYear);
+            Console.WriteLine(DateTime.Now.Day);
+            Console.WriteLine(DateTime.Today.DayOfWeek);
+            Console.WriteLine(DateTime.IsLeapYear(2019));
+
+        }
+
+
+
+        //2.Write a program to add N year(s) to the current date and display the new date.
+
+        static void NYears()
+        {
+            Console.WriteLine(DateTime.Now.AddYears(3));
+
+        }
+
+        //3.Write a program to display the date and time in a human-friendly string.
+        static void HumanFriendly()
+        {
+
+            Console.WriteLine(DateTime.Now.ToString());
```

Comment:
this is not too friendly: ToLongDateTImeString or sth like that, there is a method